extensions/wikia/SpecialHealthcheck/HealthController.class.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+/**
+ * Extra health checks controller.
+ *
+ * Currently supports requests for getting the status of database clusters
+ */
+class HealthController extends WikiaController {
+
+	private $current = '';
+
+	private $clusters = [ ];
```

Comment:
Just `[]` :smile_cat: But it's my preferred way and I couldn't find anything about it in [MW style guide](http://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP) nor in [ours](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md). I wonder if [MW style helper](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#mediawiki-php-style-helper) would change it.